source/h2_connection.c

```diff
@@ -735,16 +771,66 @@ static int s_handler_process_read_message(
     struct aws_channel_handler *handler,
     struct aws_channel_slot *slot,
     struct aws_io_message *message) {
-
-    (void)handler;
     (void)slot;
-    (void)message;
+    struct aws_h2_connection *connection = handler->impl;
+    int err;
+
+    CONNECTION_LOGF(
+        TRACE,
+        connection,
+        "id=%p: H2 connection Begin processing message of size %zu.",
+        (void *)&connection->base,
+        message->message_data.len);
+
+    if (connection->thread_data.is_reading_stopped) {
+        CONNECTION_LOGF(
+            ERROR,
+            connection,
+            "id=%p: Cannot process message because connection is shutting down.",
+            (void *)&connection->base);
+
+        aws_raise_error(AWS_ERROR_HTTP_CONNECTION_CLOSED);
+        goto shutdown;
+    }
+
+    struct aws_byte_cursor message_cursor = aws_byte_cursor_from_buf(&message->message_data);
+    err = aws_h2_decode(connection->thread_data.decoder, &message_cursor);
+    if (err) {
```

Comment:
trivial: I changed my coding style since last summer to be more consistent with everyone else's code. I was the only person doing it this way:
```
err = aws_fn();
if (err)
    error handling
```
now I do it this way like everyone else:
```
if (aws_fn())
   error handling
```